tests/steps/login.js

```diff
@@ -18,6 +18,11 @@ module.exports = () => {
       const loggedInPageSelector = 'body.apos-workflow-live-page';
 
       client.expect.element(loggedInPageSelector).to.be.present;
+
+      // do not collapse admin bar automatically
+      client.execute(function() {
+        apos.adminBar.autoCollapse = function() {};
```

Comment:
Is this change necessary - is it not possible for the tests to interact with this with the timing that a human would...?

Reply:
We can remove it. It’s to avoid problems in the future.